masu/util/__init__.py

```diff
@@ -1 +1,51 @@
-# noqa
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Masu utility methods."""
+
+import os
+
+import watchtower
+from boto3.session import Session
+
+
+def setup_cloudwatch_logging(logger):
+    """Add Cloud Watch log handler if appropriate."""
+    key_id = os.environ.get('CW_AWS_ACCESS_KEY_ID')
+    secret = os.environ.get('CW_AWS_SECRET_ACCESS_KEY')
+    if not (key_id and secret):
+        logger.info('CloudWatch logging disabled due to missing access key')
+        return
+
+    session = Session(
+        aws_access_key_id=key_id,
+        aws_secret_access_key=secret,
+        region_name=os.environ.get('CW_AWS_REGION', 'us-east-1'),
+    )
+
+    try:
+        with open('/var/run/secrets/kubernetes.io/serviceaccount/namespace', 'r') as n_file:
+            namespace = n_file.read()
+    except Exception:  # pylint: disable=broad-except
```

Comment:
Is there some reason you need this pylint disable?  

Why can't this be formulated in a way to avoid disabling pylint?

Reply:
Sure. Will do. This was just from the platform provided code.